### ISSUE TITLE
UserWarning: None of the inputs have requires_grad=True. Gradients will be None

### ISSUE THREAD

**Issue by cove1011** [shibing624/MedicalGPT#359]
学习率为零，后面的chosen和rejected全为none
<img width="415" alt="image" src="https://github.com/shibing624/MedicalGPT/assets/131332410/282008dd-abf2-44b1-be52-1f1aebc4e485">

**Comment by cove1011** [comment]
Could not estimate the number of tokens of the input, floating-point operations will not be computed

**Comment by shibing624** [comment]
use float32.
